packages/appstore/src/components/wallet-content/wallet-cfds-listing.tsx

```diff
@@ -48,21 +35,11 @@ const WalletCFDsListing = observer(({ fiat_wallet_currency = 'USD' }: TProps) =>
             </div>
         );
 
-    const { currency } = wallet_account;
     const accounts_sub_text =
         wallet_account.landing_company_name === 'svg' || wallet_account.is_virtual
             ? localize('Compare accounts')
             : localize('Account information');
 
-    const getMT5AccountAuthStatus = (current_acc_status: string) => {
-        if (current_acc_status === 'proof_failed') {
-            return 'failed';
-        } else if (current_acc_status === 'verification_pending') {
-            return 'pending';
-        }
-        return null;
-    };
-
     const is_fiat = !isCryptocurrency(currency) && currency !== 'USDT';
 
     const CryptoCFDs = () => (
```

Comment:
@thisyahlen-deriv better to exclude CryptoCFDs from this file and make a new file for that